COVID19/src/screens/registerSymptoms.js

```diff
@@ -10,68 +10,145 @@ import {
 import CustomCheck from '../components/CustomCheck';
 import Button from '../components/Button';
 import constans from '../utils/constans';
-
 import styles from '../utils/styles/symptomsStyles/symptomsStyles';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {MainFunctions} from '../utils/functions/mainFunctions';
+import axios from 'axios';
 
-DATA = [
-  {
-    id: '1',
-    name: 'Fever',
-  },
-  {
-    id: '2',
-    name: 'Cough',
-  },
-  {
-    id: '3',
-    name: 'Pain in chest with deep breaths',
-  },
-  {
-    id: '4',
-    name: 'Shortness of breaths',
-  },
-  {
-    id: '5',
-    name: 'Loss of smell',
-  },
-  {
-    id: '6',
-    name: 'Loss of taste',
-  },
-  {
-    id: '7',
-    name: 'Headache',
-  },
-  {
-    id: '8',
-    name: 'Fatigue of tiredness',
-  },
-  {
-    id: '9',
-    name: 'Muscle aches',
-  },
-  {
-    id: '10',
-    name: 'Sore throat',
-  },
-];
+var count = 0;
 
 const RegisterSymptoms = ({navigation}) => {
+  function incidentAPI(object) {
+    AsyncStorage.getItem('userToken').then((res) => {
+      axios({
+        headers: {
+          Authorization: 'Bearer ' + res,
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        method: 'post',
+        url: 'https://hidden-cliffs-21927.herokuapp.com/api/incident',
+        data: object,
+      }).then((response) => {
+        if (response.data.error === false) {
+          Alert.alert('Incident Sents', 'Alert Incident sent succesfully.', [
+            {
+              text: 'Cancel',
+              onPress: () => navigation.navigate('Main'),
+              style: 'cancel',
+            },
+            {text: 'OK', onPress: () => navigation.navigate('Main')},
+          ]);
+        } else {
+          Alert.alert('Incident Error', 'This alert was not sent', [
+            {
+              text: 'Cancel',
+              onPress: () => navigation.navigate('Main'),
+              style: 'cancel',
+            },
+            {text: 'OK', onPress: () => navigation.navigate('Main')},
+          ]);
+        }
+      });
+    });
+  }
```

Comment:
Refactor this code, mak it easy to read

Reply:
It is the distribution suggested by the writing plugin, however the "header" was adjusted to facilitate reading.